src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/GerritNotifier.java

```diff
@@ -68,6 +76,43 @@ public GerritNotifier(IGerritHudsonTriggerConfig config, GerritCmdRunner cmdRunn
         this.parameterExpander = new ParameterExpander(config, hudson);
     }
 
+    /**
+     * Notify changes with the same topic.
+     *
+     * @param event the change based event.
+     * @param command the gerrit command.
+     */
+    private void notifySameTopic(ChangeBasedEvent event, String command) {
+        Topic topic = event.getChange().getTopicObject();
+        if (topic != null) {
+            String pattern = new StringBuilder(event.getChange().getNumber()).append(',')
+                    .append(event.getPatchSet().getNumber()).toString();
+
+            if (!command.contains(pattern)) {
+                logger.error("command {} has no pattern {}", command, pattern);
+                return;
+            }
+            for (GerritServer server : PluginImpl.getServers_()) {
```

Comment:
Checking all Gerrit servers is not required, right? From my understanding so far the build started event is triggered with the corresponding server information so we just need to check whether the current server config has "isVoteSameTopic()" enabled.